### PR TITLE
Update to kubekins-test v20181218-db74ab3f4

### PR DESCRIPTION
**Pull request by ixdy**
This is a continuation of #10442, now that the upstream golang images exist.

x-ref kubernetes/kubernetes#72032

/assign @BenTheElder @krzyzacy
cc @cblecker

### CODE REVIEW

File: scenarios/kubernetes_verify.py [hunk 1]
```diff
@@ -27,12 +27,12 @@
 
 
 VERSION_TAG = {
-    '1.10': '1.10-v20180221-3655e24ae',
-    '1.11': '1.11-v20180504-30872f7f6',
-    '1.12': '1.12-v20180918-e0a9aa399',
-    '1.13': '1.13-v20181105-ceed87206',
+    '1.10': '1.10-v20181218-db74ab3f4',
```
Comment (ixdy) [inline scenarios/kubernetes_verify.py]: this might break everything.

**Comment by ixdy** [comment]
I haven't bumped `kubekins-e2e` yet since I think we want to rebase the bootstrap image on `debian:stretch`.

File: scenarios/kubernetes_verify.py (hunk 1 above)
Comment (krzyzacy) [reply scenarios/kubernetes_verify.py]: I *think* golint should work across all versions now, worth figuring out.

**Pull request merged**